edit-post/index.js

```diff
@@ -77,5 +77,6 @@ export function initializeEditor( id, post, settings ) {
 	};
 }
 
+export { default as PluginPostStatusVisibility } from './components/sidebar/plugin-post-status-visibility';
```

Comment:
I'll defer to others on the name and the slot itself cc @gziolo @mtias @aduth

Reply:
We usually export as `PluginPostStatusVisibility` (Fill) + `PluginPostStatusVisibility.Slot`, but I'm not a big fun of this approach. This one exports them as `PluginPostStatusVisibility.Fill` + `PluginPostStatusVisibility.Slot`. The only benefit of what we have in other places it that Fill is used in many places but Slot usually only in one.

Reply:
I think we should only export the Fills